doc/dev/release.md

```diff
@@ -306,11 +307,24 @@ OPERATORSUM="$(sha256sum operator-sdk | cut -d ' ' -f 1)"
 brew bump-formula-pr --strict --url=$OPERATORSDKURL --sha256=$OPERATORSUM operator-sdk
 ```
 
-Note: If there were any changes made to the CLI commands, make sure to look at the existing tests, in case they need updating.
+**Note:**  If there were any changes made to the CLI commands, make sure to look at the existing tests, in case they need updating.
 
 You've now fully released a new version of the Operator SDK. Good work! Make sure to follow the post-release steps below.
 
-### (Post-release) Updating the operator-sdk-samples repo
+### 6. Update the Upgrade version guide document
+
+1. The [Upgrade version guide][version-upgrade-guide] should be updated with the steps required for the users are able to migrate their project.
+1. Then, update the GitHub Release Notes as:
```

Comment:
This section should be merged with [this list](https://github.com/operator-framework/operator-sdk/blob/3b2f684/doc/dev/release.md#3-create-a-pr-for-post-release-version-and-changelogmd-updates).

Reply:
We have not the migration steps and the tag in step 3 as suggested. So, it cannot be done before.

Reply:
The tag will be available on release, and after adding the v1.3.0 upgrade guide section we will be able to properly reference the upgrade guide in the release notes.

Reply:
OK. I got what you mean. But then, the link will be broken until we add the steps but it is fine. I will move it to there.

Reply:
Done @estroz 👍

Reply:
The step adding the linked section to the upgrade guide should be completed before this one during a release, since we need to create an update guide before we tag a release.

Reply:
We cannot do the migration steps before the release. Note that for we be able to ensure and check them we need the tag released. So, it actually should be made or reviewed by performing them when we are upgrading the samples.